test/unit/lib/redcap_test.rb

```diff
@@ -17,15 +17,15 @@ class Redcap < ActiveSupport::TestCase
     assert_equal 5, forms.values.length
```

Comment:
I assume importing the stuff in redcap.xml is a different case than things loaded with yml files. We have no way of knowing if the parser was successful except by knowing how many things it was supposed to parse.